apb.yml

```diff
@@ -6,15 +6,20 @@ async: optional
 tags: 
   - mobile-service
 metadata:
-  displayName: AeroGear UPS
+  displayName: AeroGear Unified Push Server (UPS)
+  longDescription: "An APB deploying the AeroGear UnifiedPush Server with persistent storage and authentication through Openshift"
   imageUrl: "https://pbs.twimg.com/profile_images/1794440005/aerogear_icon-1_400x400.png"
   documentationUrl: "https://aerogear.org/push"
   providerDisplayName: "Red Hat, Inc."
-  dependencies: ['POSTGRES:95']
+  dependencies:
+    - "docker.io/aerogear/unifiedpush-wildfly:1.3.1.no-auth.Final"
+    - "docker.io/centos/postgresql-96-centos7:9.6"
+    - "docker.io/aerogear/ups-config-operator:master"
```

Comment:
Since we have the 0.0.1 tag now, should we use that instead?

Reply:
We could list that as a dependency, and actually use that.

However, since this is master, and things are in flight, and under development, I'd feel it's better to go w/ the `:master` tag.

for a real release of this APB, we will need to use real released images too ;-)

Reply:
also, it seems to be best practice to use the fqn of the image:

* https://github.com/ansibleplaybookbundle/eclipse-che-apb/blob/master/apb.yml#L14-L16
* https://github.com/ansibleplaybookbundle/mssql-apb/blob/master/apb.yml#L13
* https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/apb.yml#L9

to name only a few